docs/UserGuide.md

```diff
@@ -246,10 +246,13 @@ Format: `exit`
 Adds a client to BookKeeper. <br>
 Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
 
-<div markdown="span" class="alert alert-primary">
+<div markdown="block" class="alert alert-info">
```

Comment:
Did we have a reason for changing these

Reply:
The rest of the command uses <div markdown="block" class="alert alert-info">, hence I was thinking of keeping it consistent